content/swift/concepts/methods/methods.md

```diff
@@ -0,0 +1,69 @@
+---
+
+Title: 'Swift Methods'
+
+Description: 'Learn about methods in Swift and how they are used to perform actions and provide functionality
+
+Subjects:
+    - 'iOS' 
+    - 'Mobile Development'
+    - 'Game Development'
+
+Tags:
+    - 'Methods'
+    - 'Functions'
+    - 'Properties'
+    - 'Parameters'
+
+CatalogContent:
+- 'Learn Swift'
+- 'Build iOS Apps with SwiftUI'
+
+---
+
+## Methods
+
+In Swift, methods are like actions that an object (belonging to a class) can perform. Imagine a class as a blueprint for creating objects, and methods are the functions that these objects can execute. For example, think of a "Car" class that has a method called "startEngine." When creating a "Car" object, use the "startEngine" method to make the car's engine come to life.
```

Comment:
not just classes, can we make that edit. Also, it's great that there is an example here, though can this be two separate paragraphs? Explain methods in a abstract way and then go into examples - better transition for the reader this way

Reply:
@Wakodono classes, structs, and enums can have instance methods - can we add this change and also link out to the entries:

- https://www.codecademy.com/resources/docs/swift/classes
- https://www.codecademy.com/resources/docs/swift/structures
- https://www.codecademy.com/resources/docs/swift/enums